pybv/io.py

```diff
@@ -228,19 +272,51 @@ def _write_vmrk_file(vmrk_fname, eeg_fname, events, meas_date):
                   f'{i_dur},0', file=fout)
 
 
-def _scale_data_to_unit(data, unit):
-    """Scale `data` in Volts to `data` in `unit`."""
-    scale = SUPPORTED_UNITS.get(unit, None)
-    if scale is None:
-        msg = (f'Encountered unsupported unit: {unit}\n'
-               f'Use one of the following: {set(SUPPORTED_UNITS.keys())}')
-        raise ValueError(msg)
-
-    return data * scale
+def _scale_data_to_unit(data, units):
+    """Scale `data` in Volts to `data` in `units`."""
+    # keep track of any unsupported units to warn user
+    # these are voltage units that we convert to microvolts
+    unsupported_units = set()
+
+    # officially unsupported units that contain non-voltage
+    # related signals
+    official_unsupp_units = set()
+
+    # create a vector to multiply with to play nice with numpy
+    scales = np.zeros((len(units), 1))
+    for idx, unit in enumerate(units):
+        scale = SUPPORTED_UNITS.get(unit, None)
+        # unless the unit is 'µV', then
+        # technically it is not supported in BrainVision
+        if scale is not None and unit != 'µV':
+            unsupported_units.add(unit)
+        # if not voltage unit at all, then don't scale
+        elif scale is None:
+            official_unsupp_units.add(unit)
+            scale = 1  # don't scale the data at all
+        scales[idx] = scale
+
+    if len(unsupported_units) > 0:
+        msg = (f'Encountered unsupported voltage units: '
+               f'{", ".join(unsupported_units)}\n'
+               f'We will automatically scale and convert '
+               f'these to Volts. If this is a mistake, '
```

Comment:
This is not correct, we will not scale to Volts, but to the unit given (e.g. 'mV'). The original data must be in Volts.